.dockerignore

```diff
@@ -0,0 +1,4 @@
+venv
+.venv
+Dockerfile
```

Comment:
The `Dockerfile` should not be ignored in the `.dockerignore` file, as it is essential for building the Docker image.